src/amo/components/AddonMeta.js

```diff
@@ -54,8 +54,10 @@ export class AddonMetaBase extends React.Component {
           <p className="AddonMeta-text AddonMeta-review-count">
             {reviewCount}
           </p>
-          <Rating className="AddonMeta-Rating" rating={averageRating}
-            readOnly styleName="small" />
+          <Rating className="AddonMeta-Rating"
+            rating={averageRating}
+            readOnly
+            styleName="small" />
```

Comment:
Just a note that I think these multi-line tags look nicer/align better like:

```jsx
<Rating className="AddonMeta-Rating"
  rating={averageRating}
  readOnly
  styleName="small"
/>
```

it's actually what we (I?) have been doing recently. If eslint is cool with that I'd say change it to that while we're already changing these.